test/pages/register-a-beacon/application-complete.test.tsx

```diff
@@ -46,44 +29,82 @@ describe("ApplicationCompletePage", () => {
   });
 
   describe("getServerSideProps function", () => {
+    let mockAppContainer: IAppContainer;
+    let mockCreateRegistration: jest.Mock;
+    let mockSendGovNotifyEmail: jest.Mock;
+    let mockVerifyFormSubmissionCookieIsSet: jest.Mock;
+    let mockRedirectTo: jest.Mock;
     let context;
 
     beforeEach(() => {
-      context = { req: { cookies: { [formSubmissionCookieId]: "1" } } };
+      mockCreateRegistration = jest.fn();
+      mockSendGovNotifyEmail = jest.fn();
+      mockVerifyFormSubmissionCookieIsSet = jest.fn();
+      mockRedirectTo = jest.fn();
+
+      mockAppContainer = getAppContainerMock({
+        getCreateRegistration: jest.fn(() => ({
+          execute: mockCreateRegistration,
+        })),
+        getSendGovNotifyEmail: jest.fn(() => ({
+          execute: mockSendGovNotifyEmail,
+        })),
+        getVerifyFormSubmissionCookieIsSet: jest.fn(() => ({
+          execute: mockVerifyFormSubmissionCookieIsSet,
+        })),
+        getRedirectTo: jest.fn(() => ({
+          execute: mockRedirectTo,
+        })),
+      });
+
+      context = {
+        req: { cookies: { [formSubmissionCookieId]: "1" } },
```

Comment:
I don't think you need to pass this in now as it's all encapsulated within the dependencies we inject in via the container